modules/tensor_mechanics/include/utils/RankTwoScalarTools.h

```diff
@@ -98,12 +97,12 @@ vonMisesStress(const RankTwoTensorTempl<T> & stress)
 }
 
 /*
- * The effective strain is calculated as
+ * The effective strain increment (or equivalent von Mises strain) is calculated as
  * \epsilon_{eff} = \sqrt{\frac{2}{3}\epsilon_{ij} \epsilon_{ij}}
  */
 template <typename T>
 T
-effectiveStrain(const RankTwoTensorTempl<T> & strain)
+effectiveStrainIncrement(const RankTwoTensorTempl<T> & strain)
 {
   return std::sqrt(2.0 / 3.0 * strain.doubleContraction(strain));
```

Comment:
how do you know the strain is only the strain increment?  Is it the total strain increment or can it be elastic, plastic or creep components?   Summing this up over a simulation probably won't give the same value as the effective strain returned by the radial return method. Are we still going to return the effective strain from the radial return mapping if it is available.

Reply:
You can pass in any strain increment tensor. I guess you could also pass in stress tensor increments (or total tensors), but I'm not sure how useful the output would be in those cases.

Reply:
Is the effective strain return by a creep material that uses the radial return mapping going to be the radial return mapping computed effective strain or is it going to be the effective strain computed using this equation

Reply:
That's up to the user. For combined inelastic models and models that do not make the effective strain available it will have to be this equation.